core/src/main/scala/pureconfig/configurable/package.scala

```diff
@@ -0,0 +1,20 @@
+package pureconfig
+
+import scala.util.Try
+import java.time.{ LocalDate, LocalDateTime, LocalTime }
+import java.time.format.DateTimeFormatter
+
+package object configurable {
```

Comment:
Do you want to add a `/** ... */` comment declaring the purpose of this package? At first glance, it'd seem to be limited to date/time types, but I suspect you intend it more broadly than that.